src/sentry/rules/actions/notify_event_service.py

```diff
@@ -77,9 +78,14 @@ def after(self, event, state):
                 extra["group_id"] = group.id
                 self.logger.info("rule.fail.should_notify", extra=extra)
                 return
+            else:
+                metrics.incr("notifications.sent", instance=plugin.slug, skip_internal=False)
+                yield self.future(plugin.rule_notify)
```

Comment:
@MeredithAnya Is there any way this could throw an error? If so, we still want to make sure to send the notification on the sentry app for the reason.